src/board.c

```diff
@@ -160,29 +186,133 @@ void module_init(void) {
     break;
   }
 
-
 //	xTaskCreate( DJICanTask, (const char*)"DJI CAN",
 //	STACK_SIZE_MIN, NULL, THREAD_PRIO_HIGH, NULL );
 }
 
+// Initialize the semaphores used for intra-task communication
+void task_semaphores_init() {
+  vSemaphoreCreateBinary(onScreenDisplaySemaphore);
+  vSemaphoreCreateBinary(onMavlinkSemaphore);
+  vSemaphoreCreateBinary(onUAVTalkSemaphore);
+}
+
+/*
+void watchdog_init() {
```

Comment:
Do you use the watchdog for debugging?
If so - should we compile it conditionally?
I tend to be sceptical of commented out code since that is what git can be used for normally but for debugging functionality it seems easier to use a `#define DEBUG_WATCHDOG 1` or something and then check here, just need to change one place to reenable it